src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -32,6 +36,25 @@
  */
 class PropertyNormalizer extends AbstractObjectNormalizer
 {
+    private $allowNormalizationOfObjectsWithoutAnyProperties;
+
+    public function __construct(
+        ClassMetadataFactoryInterface $classMetadataFactory = null,
+        NameConverterInterface $nameConverter = null,
+        PropertyTypeExtractorInterface $propertyTypeExtractor = null,
+        ClassDiscriminatorResolverInterface $classDiscriminatorResolver = null,
+        callable $objectClassResolver = null,
+        array $defaultContext = [],
+        bool $allowNormalizationOfObjectsWithoutAnyProperties = false,
+    ) {
+        parent::__construct($classMetadataFactory, $nameConverter, $propertyTypeExtractor, $classDiscriminatorResolver, $objectClassResolver, $defaultContext);
+        $this->allowNormalizationOfObjectsWithoutAnyProperties = $allowNormalizationOfObjectsWithoutAnyProperties;
+
+        if (\func_num_args() < 7) {
+            trigger_deprecation('symfony/serializer', '6.1', '$allowNormalizationOfObjectsWithoutAnyProperties parameter of %s::__construct() should be explicitly provided since the default value will change to `true` in symfony/serializer >=7.0', __CLASS__);
```

Comment:
```suggestion
            trigger_deprecation('symfony/serializer', '6.2', '$allowNormalizationOfObjectsWithoutAnyProperties parameter of %s() should be explicitly provided since the default value will change to `true` in symfony/serializer >=7.0', __METHOD__);
```